dashboard/app/models/foorm/form.rb

```diff
@@ -174,9 +174,13 @@ def self.get_matrix_question_id(parent_question_id, sub_question_id)
   # by a user, as well as some additional metadata about the context
   # in which the form was submitted (eg, workshop ID, user ID).
   # @return csv text
-  def submissions_to_csv
-    filtered_submissions = submissions.
-      reject {|submission| submission.workshop_metadata&.facilitator_specific?}
+  def submissions_to_csv(submissions_to_report=nil)
```

Comment:
It might be worth updating the first part of this comment to reflect that you can now pass in an array of submissions to include

Reply:
done!